commit-message.md

```diff
@@ -0,0 +1,36 @@
+# Guidelines to create great commit messages
+
+
+We use [GitCop](https://gitcop.com) to check that commit messages are
+properly written. The rules are the following:
+
+* The first line of a commit message, called the subject line should
+  not be more than 80 characters long.
+
+* The commit message should end with the following trailers:
+
+  ```
+  Licence: MIT
+  Signed-off-by: User Name <email@address>
+  ```
+
+  where "User Name" is the author's real name and email@address one of
```

Comment:
A quick search gives non-promising results: https://www.google.com/webhp#q=pseudonymous+contribution+open+source -- seems like everyone's scared. But this isn't right. I think the right of using a pseudonyms is an important part of a free society. (e.g. http://en.wikipedia.org/wiki/The_Federalist_Papers) I will look into this, because I think the copyright concerns aren't real. Hopefully we can find some legal precedent to point to.